api/lib/services/sushi.js

```diff
@@ -156,6 +156,34 @@ const DEFAULT_REPORT_TYPE = 'tr';
 
 // https://app.swaggerhub.com/apis/COUNTER/counter-sushi_5_0_api/
 
+/**
+ * Get SUSHI URL for a given COUNTER version
+ *
+ * Some endpoints uses the (non standard) prefix `/r5` for COUNTER 5 while also using `/r51`
+ * for COUNTER 5.1. This methods removes the suffix `/r5` before adding the standard if
+ * using COUNTER 5.1
+ * It also remove the trailing `/`
+ *
+ * @param {import('@prisma/client').SushiEndpoint} endpoint - The endpoint
+ * @param {string} version - The COUNTER version
```

Comment:
`version` is optional

```suggestion
 * @param {string} [version=5] - The COUNTER version
```